app/views/jobseekers/job_applications/build/personal_statement.html.slim

```diff
@@ -17,10 +17,23 @@
       - else
         = t(".banner_no_documents_html", job_link: open_in_new_tab_link_to("‘#{job_application.vacancy.job_title}’", job_path(job_application.vacancy)))
 
+    .help-guide-container--mobile
+      h3.govuk-heading-m = t(".personal_statement_guide.title")
+      p.govuk-body = t(".personal_statement_guide.description_mobile")
+      = govuk_link_to(t(".personal_statement_guide.link_text"),
+                                                      post_path(section: "jobseeker-guides", post_name: "how-to-write-teacher-personal-statement"),
```

Comment:
I believe these should be indented once 🤷‍♂️ 

Also, it seems the font size is the same as the text above in the design.

Design: 
<img width="392" alt="Screenshot 2022-03-21 at 18 02 16" src="https://user-images.githubusercontent.com/30624173/159335836-ad13ae91-0b18-4c4e-91a3-8de819806014.png">